web/src/main/java/com/futurewei/alcor/web/entity/dataplane/InternalPortEntityNB.java

```diff
@@ -0,0 +1,22 @@
+package com.futurewei.alcor.web.entity.dataplane;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.futurewei.alcor.web.entity.port.PortEntity;
+import com.futurewei.alcor.web.entity.subnet.SubnetEntity;
+import com.futurewei.alcor.web.entity.vpc.VpcEntity;
+import lombok.Data;
+
+import java.util.Set;
+
+@Data
+public class InternalPortEntityNB extends PortEntity {
+
+  @JsonProperty("neighbor_host_ip")
+  private String neighborIp;
```

Comment:
So is it assumed that there is only one neighbor?

Reply:
changed in dcede47, as discussed yes for each port